x/concentrated-liquidity/lp_test.go

```diff
@@ -1071,12 +1070,12 @@ func (s *KeeperTestSuite) TestSendCoinsBetweenPoolAndUser() {
 		"only asset0 is greater than sender has, position creation (user to pool)": {
 			coin0:       sdk.NewCoin("eth", sdk.NewInt(100000000000000)),
 			coin1:       sdk.NewCoin("usdc", sdk.NewInt(1000000)),
-			expectError: true,
+			expectedErr: fmt.Errorf("insufficient funds"),
```

Comment:
nit: we can have a test-global definition to be reused

Reply:
Added https://github.com/osmosis-labs/osmosis/pull/5082/commits/a59c8af08ac0d831a2221f4073962fc4da2e92fa